indico/web/templates/forms/location_widget.html

```diff
@@ -0,0 +1,534 @@
+                }
+                $('.ui-dialog-content').css('overflow', 'inherit');
+
+                toggleOpenResultsList($this);
+
+                latestUsedField = $(this);
+            })
+            .on('keydown', function(e) {
+                var $this = $(this);
+                if (resultsWereOpen) {
+                    if (e.keyCode == 9) {  // on tab
+                        if ($this.val()) {
+                            // treat tab as the return key
+                            var returnKey = jQuery.Event("keydown");
+                            returnKey.which = 13;
+                            returnKey.keyCode = 13;
+                            $this.trigger(returnKey);
+                        } else {
+                            clearField($this);
+                        }
+                    } else if (e.keyCode == 13) { // on enter
+                        e.preventDefault();
+                        if ($this.val()) {
+                            if (latestSelectedOption.value == '' || latestSelectedOption.field == $this.attr('id') && latestSelectedOption.value != $this.val()) {
+                                $this.trigger('typeahead:click-on-custom-option', {item_name: $this.val()});
+                            }
+                        } else {
+                            clearField($this);
+                        }
+                    } else if (e.keyCode == 27) {  // on esc
+                        e.preventDefault();
+                        resetSelectedOption();
+                    } else if (~[38,40].indexOf(e.keyCode)) {  // on key down/up
+                        scrollResultsList($this);
+                        $this.trigger('focus.typeahead');
+                    }
+                }
+            })
+            .on('click', function() {
+                // Since focus remains on the field after selection, we need special handling for the click event
+                // in order to reopen the results list
+                Typeahead["#" + $(this).attr('id')].showLayout();
+                $(this).trigger('input');
+            });
+
+            $('.toggle-results-list').on('click', function() {
+                var container = $(this).parent().parent();
+                var input = $(container.find('.i-location-input-field')[0]);
+                if (container.hasClass('result')) {
+                    if (!input.val()) {
+                        input.trigger('typeahead:close-results-list');
+                    }
+                } else {
+                    input.trigger('focus.typeahead');
+                }
+            });
+
+            address.on('keyup', function() {
+                hiddenData.address = address.val();
+                field.val(JSON.stringify(hiddenData)).trigger('change');
+            });
+
+            $('.typeahead-field .highlighter').on('click', function() {
+                $($(this).parent().find('.i-location-input-field')[0]).trigger('focus.typeahead');
```

Comment:
$(this).parent().find('.i-location-input-field').eq(0).trigger('focus.typeahead');